cpptests/t_client.cpp

```diff
@@ -29,10 +31,18 @@ class Client {
         }
     }
 
+    Client& operator=(const Client &other_) {
+        ctx = other_.ctx;
+        return *this;
+    }
+
+    operator redisContext*() const { return ctx; }
+
     void secureConnection(const ClientSettings& settings) {
         if (redisSecureConnection(
                 ctx, settings.ssl_ca(), settings.ssl_cert(),
-                settings.ssl_key(), NULL) != REDIS_OK) {
+                settings.ssl_key(), NULL) != REDIS_OK) 
```

Comment:
Opening brace on same line as the conditional, please

Reply:
I caught it later and fixed it.
Not sure why you see it this way.